code/espurna/config/progmem.h

```diff
@@ -97,7 +97,7 @@ PROGMEM const char espurna_modules[] =
     #if RFM69_SUPPORT
         "RFM69 "
     #endif
-    #if RF_SUPPORT
+    #if RF_SUPPORT || ITEAD_SONOFF_RFBRIDGE
```

Comment:
`defined(ITEAD_SONOFF_RFBRIDGE)` as it is not 0/1 but simple token
cannot confirm, but this was an issue with ArduinoIDE build before - https://github.com/xoseperez/espurna/issues/975